formats/specialist_document/frontend/examples/utaac-decision.json

```diff
@@ -0,0 +1,53 @@
+{
+  "content_id": "3917cb0e-0928-45b7-bf63-20f9bc113a7b",
+  "base_path": "/administrative-appeals-tribunal-decisions/example-utaac-decision",
+  "title": "Example UTAAC Decision",
+  "description": "Example text.",
+  "format": "specialist_document",
+  "need_ids": [],
+  "locale": "en",
+  "updated_at": "2016-04-05T14:41:08.625Z",
+  "public_updated_at": "2016-04-05T14:41:08.000+00:00",
+  "details": {
+    "metadata": {
```

Comment:
They're not in content-store, or the examples because these three fields aren't actually sent by specialist-publisher to the publishing-api, they're only sent to rummager for search indexing.  What this means is they shouldn't be in the schema; however as there's prior art where these fields are in the schema we won't let it hold up this PR and we'll fix it later.